examples/ResizeExample.js

```diff
@@ -44,6 +44,7 @@ class ResizeExample extends React.Component {
         rowsCount={dataList.getSize()}
         onColumnResizeEndCallback={this._onColumnResizeEndCallback}
         isColumnResizing={false}
+        touchScrollEnabled={true}
```

Comment:
I tested the Resizable Columns demo with this change on:

- Chrome/Android
- Safari/iOS11
- Chrome/Mac (both dev device mode and desktop mode)
- Safari/Mac
- Firefox/Mac
- IE11/Win7
- Edge/Win10

Works correctly everywhere except IE11/Win7, where trying to load that demo gets the error:
```
		SCRIPT1014: Invalid character
		File: main.js, Line: 16952, Column: 13
```
The character referred to is the first backquote in the following line in index.js:
```
      type: `webpack${type}`,
```